package/kedro_viz/server.py

```diff
@@ -113,6 +127,43 @@ def run_viz(port=None, line=None) -> None:
     display(HTML(wrapper))
 
 
+def _load_from_file(load_file: str) -> dict:
+    global data  # pylint: disable=global-statement,invalid-name
+    data = json.loads(Path(load_file).read_text())
+    for key in ["nodes", "edges", "tags"]:
+        if key not in data:
+            raise KedroCliError(
+                "Invalid file, top level key '{}' not found.".format(key)
+            )
+    return data
+
+
+def _get_pipeline_from_context(context, pipeline_name):
+    if match(kedro.__version__, ">=0.15.2"):
+        if pipeline_name:
```

Comment:
Is there any point in checking `pipeline_name`? `_get_pipeline()` knows how to handle the `None`s.

Reply:
If we indeed don't need it, the `else` will go away too, but keep the if-raise and final return.

Reply:
Thanks! I simplified the checking and removed `else` :)